fix: improve git PAT configuration documentation (#2567)
codenvy/che-docs: modules/end-user-guide/pages/using-a-git-provider-access-token.adoc

+
* GitHub: link:https://docs.github.com/en/rest/users/users#get-a-user[Get a user]. See the `id` value in the response.
* GitLab: link:https://docs.gitlab.com/ee/api/users.html#for-normal-users[List users: For normal users], use the `username` filter: `/users?username=:username`. See the `id` value in the response.

> * Bitbucket Server: use the `curl` command: `curl -D- -u username:password -X GET -H "Content-Type: application/json" https://{baseurl}/rest/api/latest/users/{userSlug}` and see the `id` value in the response. More information could be found in the link:https://developer.atlassian.com/server/bitbucket/rest/v802/api-group-api/#api-api-latest-users-userslug-get[Get users BitBucket documentation] and the link:https://developer.atlassian.com/server/bitbucket/how-tos/example-basic-authentication/[authentication for BitBucket API].

Reviewer: @svor,  in `https://{baseurl}/rest/api/latest/users/{userSlug}`,
`{baseurl}` and `{userSlug}` are some environment variables or are these placeholders for some values that the reader has to enter here manually?

Author: @max-cx these placeholders for the user's input
It was taken from https://developer.atlassian.com/server/bitbucket/rest/v802/api-group-api/#api-api-latest-users-userslug-get

Reviewer: Thank you.

> * Bitbucket Server: use the `curl` command: `curl -D- -u username:password -X GET -H "Content-Type: application/json" https://{baseurl}/rest/api/latest/users/{userSlug}` and see the `id` value in the response. More information could be found in the link:https://developer.atlassian.com/server/bitbucket/rest/v802/api-group-api/#api-api-latest-users-userslug-get[Get users BitBucket documentation] and the link:https://developer.atlassian.com/server/bitbucket/how-tos/example-basic-authentication/[authentication for BitBucket API].

Reviewer: @svor, thank you.

1. I haven't seen a response, so I'm assuming that here I use my user slug value for the API call, and then I locate the `id` field in the response, correct?

2. Looks like our repo tools are failing on Atlassian's markup.
So I wonder if we can switch to our habitual markup here, and also some suggested changes to the wording:

```suggestion
* Bitbucket Server: Get the `id` by running `curl -D- -u username:password -X GET -H "Content-Type: application/json" https://__<baseurl>__/rest/api/latest/users/__<userSlug>__`. See the Bitbucket API documentation about link:https://developer.atlassian.com/server/bitbucket/rest/v802/api-group-api/#api-api-latest-users-userslug-get[using the user slug] and link:https://developer.atlassian.com/server/bitbucket/how-tos/example-basic-authentication/[authentication].
```

Author: 1. Correct
2. Looks good, applying your suggestion, thanks!

<1> Your {prod-short} user ID.
<2> The Git provider name: `github` or `gitlab` or `bitbucket-server` or `azure-devops`.
<3> The Git provider URL.

> <4> Your Git provider user ID wrapped with `' '` or `" "`.

Reviewer: ```suggestion
<4> Your Git provider user ID wrapped in `' '` or `" "`.
```

> <4> Your Git provider user ID, the value should be wrapped with ' ' or " ".

Reviewer: ```suggestion
<4> Your Git provider user ID wrapped with `' '` or `" "`.
```
